### PR TITLE
Some minor changes to the customer.json schema.

### PR DESCRIPTION
**Pull request by igorkosta**
@agalloch, @gsamokovarov
Please review

### CODE REVIEW

**Comment by schorsch** [comment]
can you check the failing spec? Not sure if it is a false-positve from one of the schema validators

**Comment by agalloch** [comment]
Where would we store `"additional_first_name", "occupation", "birthplace"` and `"own_interest"` attributes? Extend the existing models?

File: schema/v1.0/customer.json [hunk 1]
```diff
@@ -48,6 +73,21 @@
       "type"        : "string",
       "maxLength"   : 30
     },
+    "occupation"  : {
+      "description" : "Customer's current occupation: 1 - employee, 2 - executive, 3 - self-employed, 4 - freelancer, 5 - student, 6 - trainee, 7 - retired, 8 - privatier, 9 - unemployed",
```
Comment (agalloch) [inline schema/v1.0/customer.json]: `privatier` - A pirate? : ) What is the English equivalent?

File: schema/v1.0/customer.json [hunk 2]
```diff
@@ -19,11 +18,32 @@
...
+      "description" : "The additional given name of the customer",
+      "type"        : "string",
+      "maxLength"   : 30
+    },
     "last_name" : {
       "description" : "The family name of the customer",
       "type"        : "string",
       "maxLength"   : 30
     },
+    "password"   : {
```
Comment (schorsch) [inline schema/v1.0/customer.json]: should we call it new_password, so the name reflects the pure setter intention?

Comment (agalloch) [inline schema/v1.0/customer.json]: But then again, if I'm a user who is about to create their new customer and I encounter an attribute called `new_password`, I'd be wondering "now should there be an old_password, what am I missing?" It's a `new_password` only if you PUT to your existing customer with it : )

Comment (igorkosta) [inline schema/v1.0/customer.json]: @schorsch @agalloch was exactly my thought.

**Pull request merged**